keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/unary/ReshapeVertex.java

```diff
@@ -13,6 +14,7 @@
 
 public class ReshapeVertex extends DoubleUnaryOpVertex implements Differentiable {
 
+    @ExportVertexToPythonBindings
```

Comment:
Not saveable?

Reply:
Not at the moment - requires more save functionality